### PR TITLE
Testing To Do list part 1

### PR DESCRIPTION
**Pull request by Charzy-1**
Used jest framework for testing.
• Created a test index.test.js) for a file containing the add item and delete item functions

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  transform: {
+    '^.+\\.jsx?$': ['babel-jest', { configFile: './.babelrc' }],
+  },
+  moduleFileExtensions: ['js', 'jsx'],
+  moduleDirectories: ['node_modules', 'src'],
+  testEnvironment: 'jest-environment-jsdom',
```
Comment (basitali111) [inline jest.config.js:7]: Add this line to this file to make your project run without webpack errors as in this picture

Screenshot (302)

 so the app can focus on adding the test

**Review by basitali111 — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @Charzy-1 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!




## Highlights :green_circle: 


- PR is descriptive and professional :+1: 
- Jest was correctly utilized for authentication.
- No Linter error :heavy_check_mark: 


## Required Changes ♻️
- Please make sure your tests are passing before you submit your project. here is the error message and the details about how you are going to fix it. Fix your code and finally check by running npm test to check if it passes.
 
Screenshot (304)

#Hint: 
Here's how you can resolve this issue:

* Install identity-obj-proxy: This package can be used to mock CSS imports in Jest tests.
npm install --save-dev identity-obj-proxy

* Update Jest configuration: You need to update the Jest configuration to use identity-obj-proxy for CSS module imports. You can do this in the jest.config.js file or directly in your package.json.

* Verify Babel configuration: Make sure your Babel configuration in .babelrc or babel.config.js is correct. It should include presets like @babel/preset-env.

Screenshot (303)


_Check the comments under the review._

## Optional suggestions



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy] have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can contact us via._

**Comment by Charzy-1** [comment]
I have done the necessary configuration to ensure the test are passing when run. Please check @basitali111

**Review by Pamphilemkp — APPROVED** [review]
Hi @Charzy-1 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅  
- All the tests are passing
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @Pamphilemkp

------
_As described in the [Code reviews limits policy]()  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form]()._

**Pull request merged**